ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/DdlDmlRequestValidators.java

```diff
@@ -22,7 +22,7 @@
 
 final class DdlDmlRequestValidators {
 
-  private static final String QUOTED_STRING_OR_SEMICOLON = "(`([^`]*|(``))*`)|('([^']*|(''))*')";
+  private static final String QUOTED_STRING = "(`([^`]*|(``))*`)|('([^']*|(''))*')";
```

Comment:
I'm confused. Is this saying that strings are allowed to be enclosed in backticks? I thought only single quotes were allowed for string literals.

Reply:
Backquoted identifiers can have semicolons in them.

Reply:
Got it. Can we rename this variable to `QUOTED_STRING_OR_IDENTIFIER` in that case?